apps/ledger-live-mobile/src/contentCards/layouts/grid/utils.tsx

```diff
@@ -0,0 +1,39 @@
+import { useTheme } from "styled-components/native";
+
+export const useItemStyle = (
+  isNotSingleCard: boolean,
+  isFirstItem: boolean,
+  isLastItem: boolean,
+) => {
+  const { colors } = useTheme();
+
+  const stylesMap = {
+    singleCard: {
+      borderRadius: 12,
+    },
+    firstStackCard: {
+      borderTopLeftRadius: 12,
+      borderTopRightRadius: 12,
+    },
+    lastStackCard: {
+      borderBottomLeftRadius: 12,
+      borderBottomRightRadius: 12,
+      borderTopWidth: 1,
+      borderTopColor: colors.opacityDefault.c10,
+      borderBottomColor: colors.opacityDefault.c10,
+    },
+    middleCards: {
+      borderTopWidth: 1,
+      borderTopColor: colors.opacityDefault.c10,
+      borderBottomColor: colors.opacityDefault.c10,
+    },
+  };
+
+  if (!isNotSingleCard) return stylesMap.singleCard;
```

Comment:
Can we change this parameter to `isSingleCard` to avoid the double negation please ?

Reply:
✅